src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
+@import "./variables";
+
+* {
+  margin: 0;
+}
```

Comment:
you add `margin: 0` to all elements on the page, do you really need it?

learn more about css selectors - [link](https://mate.academy/learn/css-basics/css-selectors?section=video&videoId=4330)